src/components/EditImageModal/index.jsx

```diff
@@ -342,29 +417,78 @@ export default class EditImageModal extends React.Component {
   getImage = () => (
     <img
       alt=""
+      className={classNames(styles['image-preview-image'], { invisible: !this.state.isImageLoaded })}
       src={this.getImageAssetSource()}
       onLoad={this.onImageLoad}
       onError={this.onImageError}
       ref={(input) => { this.imgRef = input; }}
     />
   );
 
+  /*
+   image is conditionally displayed so that onImageError is not called on initial
+    modal open, when image source is empty string
+  */
   getImagePreviewPlaceholder = () => (
-    <div className="image-preview"><WrappedMessage message={messages.editImageModalImagePreviewText} /></div>
+    <div className={styles['image-preview-placeholder']}>
+      <WrappedMessage message={messages.editImageModalImagePreviewText} >
+        {displayText =>
+          (<span className={classNames({ invisible: this.state.isImageLoaded })}>
```

Comment:
We hide the "Image Preview" text inside of the placeholder div if an image is loaded over top of it, to avoid it peeking out from underneath.